js-intermediate.test.js

```diff
@@ -24,3 +30,7 @@ describe("greeter", () => {
     expect(greeter(person3)).toEqual("Hello, Pop!")
   })
 })
+const greeter = (person) => {
+  return `Hello, ${person}!`
+}
+console.log(greeter());
```

Comment:
Remember that when coding in a test file you don't need to log your function. The test will handle that for you.